test/test_container.py

```diff
@@ -560,3 +550,43 @@ class AnonymousService():
         'Service class must define a `name` attribute '
         '(test.test_container.AnonymousService)'
     )
+
+
+@pytest.mark.parametrize("args,kwargs", [
+    ((True,), {}),
+    ((), {'protected': True})
+])
+def test_spawn_managed_thread_backwards_compat_warning(
+    warnings, container, args, kwargs
+):
+
+    def wait():
+        Event().wait()
+
+    container.spawn_managed_thread(wait, *args, **kwargs)
+
+    # TODO: pytest.warns is not supported until pytest >= 2.8.0, whose
+    # `testdir` plugin is not compatible with eventlet on python3 --
+    # see https://github.com/mattbennett/eventlet-pytest-bug
```

Comment:
fyi i think the fix is https://github.com/eventlet/eventlet/pull/257 . trying to push that along. (my journey's in https://github.com/eventlet/eventlet/pull/314)